src/client/hud.cpp

```diff
@@ -278,6 +278,21 @@ void Hud::drawItems(v2s32 upperleftpos, v2s32 screen_offset, s32 itemcount,
 	}
 }
 
+#define calculateScreenPos()		\
```

Comment:
it's very ugly to use that as a define

Reply:
Well, how about an inline function? I essentially needed a goto at this point to avoid code duplication.

Reply:
An inline function would be preferred

Reply:
Solved. Didn't use an inline function or goto though, created a proper function instead.